donations/models.py

```diff
@@ -66,10 +66,10 @@ def serve(self, request, *args, **kwargs):
         donor_address_form = DonorAddressForm(request.POST)
         donation_form = DonationForm(request.POST)
 
-        if request.method == "POST":
-            if donation_form.is_valid():
-                return process_donation_request(
-                    request, donation_form, donor_address_form
+        if request.method == "POST" and donation_form.is_valid():
+           
+            return process_donation_request(
+                request, donation_form, donor_address_form
```

Comment:
Please lint these changes with `pre-commit` or `black`

Reply:
I did use `black` and commited the code again. Please let me know if everything is fine.